From 06345526875e5c9d4bcd16f44dfe6ca9d0188aaf Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 12 Oct 2018 10:41:46 +0300
Subject: [PATCH] version-hooks: differentiate between hl devices

Current version-hooks implementation is just looks for the latest
annotated tag and uses it for version creation.
Since we have more than one devices and would need to separate
the releases/versions for each of the devices, we need to separate
the tags and create version according to device specific tag.
Adjust the 'git describe' call to be aware of the above facts and
produce device specific versions.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
index aa41bd9..c04c7ba 100644
--- a/package/version-hooks/version-hooks.mk
+++ b/package/version-hooks/version-hooks.mk
@@ -1,7 +1,8 @@
 define HL_VERSION_GEN
 	if [ -n $(SRCDIR) ]; then \
 		pushd $(SRCDIR); \
-		echo -n "-`git describe --dirty 2>/dev/null`" > $(@D)/localversion-habanalabs; \
+		echo -n "-`git describe --dirty --match="hl-$(HL_DEV)-*" 2>/dev/null`" \
+			> $(@D)/localversion-habanalabs; \
 		popd; \
 	fi
 endef
